userguide/plugins.rst

```diff
@@ -389,40 +389,93 @@ describes each required entry.
-   +-------------------------+--------------------------------------------------------------------+
-   | :samp:`"artifact":`     | URL of the plugin artifact repo.                                   |
-   |                         |                                                                    |
-   +-------------------------+--------------------------------------------------------------------+
-   | :samp:`"pkgs":`         | Port of the plugin.                                                |
-   |                         |                                                                    |
-   +-------------------------+--------------------------------------------------------------------+
-   | :samp:`"packagesite":`  | CDN the plugin jail uses. Default for the TrueOS CDN is            |
-   |                         | http://pkg.cdn.trueos.org/iocage .                                 |
-   |                         |                                                                    |
-   +-------------------------+--------------------------------------------------------------------+
-   | :samp:`"fingerprints":` | :samp:`"function":`                                                |
-   |                         |                                                                    |
-   |                         | Default is *sha256*.                                               |
-   |                         |                                                                    |
-   |                         | :samp:`"fingerprint":`                                             |
-   |                         |                                                                    |
-   |                         | The pkg fingerprint for the artifact repo. Default is              |
-   |                         | *226efd3a126fb86e71d60a37353d17f57af816d1c7ecad0623c21f0bf73eb0c7* |
-   |                         |                                                                    |
-   +-------------------------+--------------------------------------------------------------------+
-   | :samp:`"official":`     | Defines if this an official iXsystems supported plugin.            |
-   |                         | Enter *true* or *false*.                                           |
-   |                         |                                                                    |
-   +-------------------------+--------------------------------------------------------------------+
+   +---------------------------+--------------------------------------------------------------------+
+   | Data Field                | Description                                                        |
+   +===========================+====================================================================+
+   | :samp:`"name":`           | Name of the plugin.                                                |
```

Comment:
These should be :literal:, because they are literal text, not samples of what the user should type.